### ISSUE TITLE
As a front end user, I want a filter documents by date so that I can find documents known to be from a particular time period.

### ISSUE THREAD

**Issue by rlskoeser** [Princeton-CDH/geniza#865]
## dev notes
- [ ] index standardized date as date range in solr
- [ ] add date range filter to form
- [ ] display date range on search page
- [ ] add date range filtering to view

**Comment by rlskoeser** [comment]
@richmanrachel I think you're not seeing the latest styles, would you try a hard refresh and see if the design looks more usable?  (We may still need additional instructions)

**Comment by richmanrachel** [comment]
@rlskoeser - also, if we're not using the filter to include dates in other calendar types, can we make the full dates searchable (so you can put 4963 into the search to get the AM docs dated to that year)?

**Comment by rlskoeser** [comment]
@gissoo could you advise? Your designs had "From Year" and "To Year" as placeholder text, but we can't do non-numeric placeholders for a numeric input. Would it work to add labels to the left of the inputs? 

@richmanrachel would it work to make the label **Document Dates (CE)**, or does the CE need to be elsewhere? Also, should it be **Document Dates** or **Document Date** ?

**Comment by rlskoeser** [comment]
@richmanrachel yes! I think that would be covered by the date searching we discussed yesterday, I hadn't gotten around to creating an issue yet. Great to have a specific use case.

**Comment by richmanrachel** [comment]
@rlskoeser - having Document Dates (CE) as the title makes sense to me. I think the plural is fine since you're searching for multiple documents most of the time? I'll double check with Marina

**Comment by rlskoeser** [comment]
Yes, you're filtering on multiple documents but each one only has one date; I'm not sure semantically which makes more sense.

**Comment by gissoo** [comment]
Just noticing some things here for usability / potentially problematic:

- whenever there is an alert/invalid message the fields jump and I'm selecting the "has image" filter by mistake. 
- it's not a great experience that when you change the "from" date to a value that exceeds the "to" date you instantly get an "invalid" message – could it update the "to" date to match the "From" date by default? Or just reset the "to" date field when you revise the "from" date? 

- ![Screen Shot 2022-05-24 at 10 48 59 AM](https://user-images.githubusercontent.com/44615389/170065033-956b1990-f5d8-4995-b9c7-892417e3b1d2.png)

**Comment by rlskoeser** [comment]
@gissoo I'm disinclined to add extra logic because it could introduce false assumptions or unnecessary assumptions (e.g., it could make as much sense to clear out the other value as to increase it), and also because it increases the complexity of the code. 

The jump in fields is not great and should probably be addressed; hopefully there's a simple way to adjust the spacing so that the error message space is always accounted for.

**Comment by richmanrachel** [comment]
@rlskoeser - I love it! Closing.
